providers/cloudflare/resources/cloudflare_zones.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+package resources
+
+func (c *mqlCloudflareZone) id() (string, error) {
```

Comment:
this is not going to work, since you are also discovering this type as individual asset. If you want to be able to do this, you need to add an init function. For example https://github.com/mondoohq/cnquery/blob/main/providers/gcp/resources/bigquery.go#L202-L250

Reply:
not going to work how?
It's included into auto-generated files, so what it's needed for atm is that zones are making separate resources

Reply:
updated